### PR TITLE
chore: Remove the pull request template

### PR DESCRIPTION
**Pull request by PolariTOON**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1 +0,0 @@
-- [ ] Changelog updated if needed
```
Comment (trollepierre) [inline .github/pull_request_template.md:1]: @paultranvan : is it ok for you, to remove that?

I added it, after we notice together, one of your PR didn't update changelog.

Comment (JF-Cozy) [reply .github/pull_request_template.md:1]: y'a un sujet en front à traiter sur ce changelog, faut absolument qu'on en parle mardi. En attendant on peut supprimer je pense.

**Review by JF-Cozy — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (paultranvan) [reply .github/pull_request_template.md:1]: Why do you want to remove it? Do we still need to manually update the changelog?

**Review by trollepierre — CHANGES_REQUESTED** [review]
waiting for changelog discussion

**Pull request closed without merging**